src/api/Rakefile

```diff
@@ -29,7 +29,7 @@ unless Rails.env.production?
   require 'haml_lint/rake_task'
 
   RuboCop::RakeTask.new(:rubocop) do |task|
-    task.options = ['-D', '-F', '--fail-level', 'convention']
+    task.options = ['-D', '-a']
```

Comment:
This looks unrelated. If this was an intended change, please move it to another PR.